services/web/server/src/simcore_service_webserver/projects/_common_models.py

```diff
@@ -13,8 +13,8 @@
 
 
 class RequestContext(BaseModel):
-    user_id: UserID = Field(..., alias=RQT_USERID_KEY)
-    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)  # type: ignore[literal-required]
```

Comment:
Same here. This ignore might be avoidable. Note that the exact same expression is in other parts of the code and were not ignored there. Probably some reference or mypy config is missig here